ckanext/saml2/plugin.py

```diff
@@ -653,3 +660,18 @@ def slo(self):
     def staff_login(self):
         """Default login page for staff members."""
         return self.login()
+
+    def edit(self, id=None, data=None, errors=None, error_summary=None):
+        c = p.toolkit.c
+        c.allow_user_changes = config.get('ckan.saml2.allow_user_changes', False)
```

Comment:
not consistent behavior - you'll get string, when this directive defined and boolean, when it does not. Use converters to boolean